reportTypes.md

```diff
@@ -0,0 +1,61 @@
+## Report and data types
```

Comment:
See that the documentation format looks a lot like open api spec. Maybe consider enum description for strings that can be one of a set of predefined strings? Like status, publicationCategory, channelType and channelLevel?